plugins/TwoFactorAuth/vue/src/LoginTwoFactorAuth/LoginTwoFactorAuth.vue

```diff
@@ -0,0 +1,180 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock :content-title="translate('TwoFactorAuth_TwoFactorAuthentication')">
+    <div class="message_container">
+      <FormErrors
+        :form-errors="formData.errors"
+      />
+
+      <Notification
+        v-if="accessErrorString"
+        :noclear="true"
+        context="error"
+      >
+        <strong>{{ translate('General_Error') }}</strong>:
+        <span v-html="$sanitize(accessErrorString)" />
+        <br/>
+      </Notification>
+    </div>
+
+    <form v-bind="formDataAttributes" class="loginTwoFaForm">
+      <div class="row">
+        <div class="col s12 input-field">
+          <input
+            type="hidden"
+            name="form_nonce"
+            id="login_form_nonce"
+            :value="formNonce"
+          />
+
+          <input
+            type="text"
+            name="form_authcode"
+            placeholder=""
+            id="login_form_authcode"
+            class="input"
+            value=""
+            size="20"
+            autocorrect="off"
+            autocapitalize="none"
+            autocomplete="off"
+            tabindex="10"
+            autofocus="autofocus"
+          />
+
+          <label for="login_form_authcode">
+            <i class="icon-user icon"></i>
+            {{ translate('TwoFactorAuth_AuthenticationCode') }}
+          </label>
+        </div>
+      </div>
+
+      <div class="row actions">
+        <div class="col s12">
+          <input
+            class="submit btn"
+            id="login_form_submit"
+            type="submit"
+            :value="translate('TwoFactorAuth_Verify')"
+            tabindex="100"
+          />
+        </div>
+      </div>
+
+    </form>
+
+    <p>
+      {{ translate('TwoFactorAuth_VerifyIdentifyExplanation') }}
+      <span v-html="learnMoreText" />
+
+      <br /><br />
+      <strong>{{ translate('TwoFactorAuth_DontHaveYourMobileDevice') }}</strong>
+      <br />
+      <a href="https://matomo.org/faq/how-to/faq_27248" rel="noreferrer noopener">
```

Comment:
Even though this wasn't the case before, I would suggest to give that one a `target="_blank"`
```suggestion
      <a href="https://matomo.org/faq/how-to/faq_27248" rel="noreferrer noopener" target="_blank">
```

Reply:
:+1: